tests/integration/testHelpers.ts

```diff
@@ -443,10 +443,10 @@ export const skipIfMainnet = (() => {
   let chainId: number
   return async (testContext: Mocha.Context) => {
     if (!chainId) {
-      const { parentChain } = await testSetup()
-      chainId = parentChain.chainID
+      const { childChain } = await testSetup()
```

Comment:
The only usage of `parentChain` from `testSetup` was to detect if it was mainnet, so we check the child